src/components/Application/ApplicationContainer.tsx

```diff
@@ -1,26 +1,29 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
 import { Container, Header, Icon, Label } from 'semantic-ui-react'
-import strings from '../../utils/constants'
+import { useRouter } from '../../utils/hooks/useRouter'
 import { TemplateDetails, User } from '../../utils/types'
-
-export interface AppHeaderProps {
+import strings from '../../utils/constants'
+export interface ApplicationContainerProps {
   template: TemplateDetails
   currentUser: User | null
 }
 
-const ApplicationContainer: React.FC<AppHeaderProps> = ({ template, currentUser, children }) => {
+const ApplicationContainer: React.FC<ApplicationContainerProps> = ({
+  template,
+  currentUser,
+  children,
+}) => {
+  const { replace } = useRouter()
   const { code, name } = template
   return (
-    <Container id="application-area">
+    <Container>
```

Comment:
Why remove the id? I know there was no longer any CSS referring to it, but it's still helpful to have ids for thing, especially major structural components like this. (Makes it a lot easier seeing what's what in the Inspector and relating it back to the code).

Reply:
NP. Putting this back.